docs/claims.md

```diff
@@ -16,10 +16,10 @@ If you are making a claim on Polkadot for the first time, please read on below i
 [making a claim](#making-a-claim) section. If you've already claimed during the pre-genesis claims
 period, please proceed to the [attesting to a statement](#attesting-to-a-statement) section instead.
 
-> To learn more on How to claim your DOTs post genesis, check out our walkthrough
+> To learn more on How to claim your DOT post genesis, check out our walkthrough
 > [video](https://www.youtube.com/watch?v=rjhWfKXJTCg&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=22)
 > and our video on
-> [Claiming DOTs with an Ethereum address generated using an old mnemonic phrase](https://www.youtube.com/watch?v=AlwrM27x3As&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=16)
+> [Claiming DOT with an Ethereum address generated using an old mnemonic phrase](https://www.youtube.com/watch?v=AlwrM27x3As&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=16)
```

Comment:
We should also let the community team know that we've switched from DOTs to DOT, so that they can update the technical explainer videos